pypyr/steps/shell.py

```diff
@@ -33,7 +44,12 @@ def run_step(context):
     # spaces to allow for natural shell language input string.
     logger.debug(f"Executing command string: {context['cmd']}")
 
+    interpolated_string = pypyr.format.string.get_interpolated_string(
+        input_string=context['cmd'],
+        context=context)
+    logger.debug(f"Interpolated string: {interpolated_string}")
```

Comment:
I find these logging entries slightly misleading, in that "Interpolated string" is the _actual_ command that's executed.  The previous log entry logs the pre-formatted or unformatted command.  I'd make the second one say "Executing command: {interpolated_string}" and the first one say something like "Processing command string: {context['cmd']}"